pkg/nsx/services/builder_test.go

```diff
@@ -406,7 +406,7 @@ func TestValidateSelectorExpressions(t *testing.T) {
 	assert.Equal(t, 2, totalCriteriaCount)
 	assert.Equal(t, 30, totalExprCount)
 
-	// Case: total count of expressions exceed NSX limit '15' in one criterion mixed criteria
+	// Case: total count of expressions exceed NSX limit '15' in one criteria mixed criteria
```

Comment:
Maybe "in one criteria with mixed member type"

Reply:
lol， change it back.